Libs/MRML/DisplayableManager/vtkMRMLCameraWidget.h

```diff
@@ -164,6 +166,7 @@ class VTK_MRML_DISPLAYABLEMANAGER_EXPORT vtkMRMLCameraWidget : public vtkMRMLAbs
 
   double MotionFactor;
   double MouseWheelMotionFactor;
+  bool stabilize3D = false;
```

Comment:
Initialization of variables are inconsistent. Either initialize all the other variables here or initialize all the variables in the constructor.

Reply:
Initialized in the constructor now.